addons/mail/static/src/models/composer/composer.js

```diff
@@ -601,7 +601,9 @@ function factory(dependencies) {
          */
         _getCommandFromText(content) {
             if (content.startsWith('/')) {
-                return content.substring(1).split(/\s/)[0];
+                const command_name = content.substring(1).split(/\s/)[0];
+                const command = this.env.messaging.commands.find(command => command.name === (command_name));
+                return command ? command.name : undefined;
```

Comment:
As per discussed, make the method actually return the command, and properly pass the command.name to the rpc.